web/src/components/documents/NutrientContainer.vue

```diff
@@ -42,14 +46,65 @@
   };
 
   const loadMultiple = async () => {
-    const documentResponse = await filesService.generatePdf(pdfStore.documents);
+    const groupedDocs = pdfStore.groupedDocuments;
+    const allDocs: StoreDocument[] = [];
+    Object.values(groupedDocs).forEach((userGroup: any) => {
```

Comment:
nit: not sure what side of the fence this team falls on with :any, but personally I prefer types - especially in unstable code bases, as it helps future developers track references to fields that need to be maintained.